app/src/main/res/values/strings.xml

```diff
@@ -2579,4 +2579,12 @@ Please check the URL again."</string>
     <string name="text_to_search">Text to search</string>
     <string name="myprojects_list_menu_title_restore_projects">Restore Projects</string>
     <string name="sort_projects">Sort projects</string>
+    <string name="code_shrinker_use_r8">Use R8</string>
+    <string name="activated">Activated</string>
```

Comment:
prefix still missing

Reply:
activated doesn't necessarily denote anything specific related to code shrinking and can definitely be used in other places

Reply:
can but isn't yet

Reply:
Still it won't make sense to include that prefix for a string that's not directly related to it, that would be like using `alert_dialog_button_ok_text` id for string `OK`

Reply:
meanwhile `android.R.string.ok` lol

this is pretty 🤓y to argue about so whatever